.github/workflows/label.yml

```diff
@@ -14,6 +14,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/labeler@v2
-      with:
-        repo-token: "${{ secrets.MY_ACTIONS_TOKEN }}"
+      - uses: actions/labeler@v2
+        with:
+          repo-token: "${{ secrets.MY_ACTIONS_TOKEN }}"
+      - uses: TimonVS/pr-labeler-action@v3.1.0
+        with:
+          configuration-path: .github/pr-labeler.yml # optional, .github/pr-labeler.yml is the default value
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This token works?

I created my own because it seemed that this one wasn't working

Reply:
Yep you can see that the bot added the label to this PR